### PR TITLE
make jspm friendly

### PR DESCRIPTION
**Pull request by dtm5011**
Remove .gitignore file on release so that the repo can be checked into other repos.
Set the main in package.json to the correct place.

### CODE REVIEW

File: bin/release.sh [hunk 1]
```diff
@@ -9,8 +9,10 @@ git checkout -b build
 npm run clean
 npm run build
 
+git rm .gitignore
+
 # Create git tag, which is also the Bower/Github release
-git add dist -f
+git add .
```
Comment (roryf) [inline bin/release.sh]: This could add undesired build artefacts, is there a reason it can't just be `dist`?

File: package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "name": "ng-cookies",
   "version": "1.0.2",
   "description": "Simple cookie service for AngularJS",
-  "main": "index.js",
+  "main": "/dist/ng-cookies.min.js",
```
Comment (roryf) [inline package.json]: This breaks in a commonjs/npm environment, is there another property that jspm can use or can it bundle the `index.js` directly?

**Comment by dtm5011** [comment]
made some changes

File: package.json (hunk 2 above)
Comment (roryf) [inline package.json]: this still needs reverted

Comment (dtm5011) [inline package.json]: fixed

**Comment by roryf** [comment]
:+1:

**Pull request merged**